doc/cephadm/operations.rst

```diff
@@ -402,7 +402,9 @@ identifying anomalies.
 
 Client keyrings and configs
 ===========================
-
+By default, starting from versions 16.2.10 and 17.2.1 in addition to
+the default location ``/etc/ceph/`` ceph also stores its configuration
```

Comment:
```suggestion
the default location ``/etc/ceph/`` cephadm also stores its config and keyring
```
I feel like we should specify here this is a cephadm feature. The rest of ceph is still unaware of this config in /var/lib/ceph/<fsid> and would only check /etc/ceph (if it checks for the file at all)

Reply:
Suggest also including a bit about why this is being done and how the new per-fsid copy will be used differently.

Reply:
Groovy.  I'm thinking especially about installations that have existing automation to maintain `ceph.conf` that might run afoul of this.  I suspect that it might be akin to the way that in a previous life we used `ceph-deploy` from an admin node, and had to carefully keep `ceph.conf` in sync there with the copies in `/etc/ceph` on the daemon nodes.

Reply:
Ack.  So it's possible then to manage more than one cluster from a given `cephadm` node?  That mirrors the situation in the abovementioned previous life, where initially we had one admin node per DC with potentially multiple clusters, and would have to cd into a cluster-specific subdirectory before running `ceph-deploy`.  This was awkward and prone to error, so we switched to a dedicated `ceph-deploy` node per cluster.

Reply:
Not sure about `ceph-deploy` (as I'm not familiar with the tool) but with `cephadm` things are different and you can specify which cluster you want to manage/operate (by specifying the fsid). In case `cephadm` detects multiple cluster on the host it will force the user to specify which of them he wants to manage. AFAIK there's no risk to mess other clusters running in the same host because each cluster has its own daemons and all the operations are performed from the `cephadm shell` which is runs everything per fsid and by using the corresponding cluster admin key.